src/main/java/core/basesyntax/dao/ConverterDataToObjImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.FruitTransaction;
+import java.util.List;
+
+public class ConverterDataToObjImpl {
+    private static final String SEPARATOR = ",";
+    private static final int OPERATION_SEQUENCE = 0;
+    private static final int FRUIT_NAME_SEQUENCE = 1;
+    private static final int FRUIT_QUANTITY_SEQUENCE = 2;
+
+    public List<FruitTransaction> convertAll(List<String> listOfStrings) {
+        List<FruitTransaction> list = listOfStrings.stream()
```

Comment:
don't name data structures, or containers by data type - map map, list list, []array - name them after what they store. It's your opportunity to increase readability